### ISSUE TITLE
Update repo with newest code and documentation

### ISSUE THREAD

**Issue by jsejcksn** [mrmartineau/xkcd-api#6]
[xkcd.now.sh](https://xkcd.now.sh/) currently returns the following:

> # XKCD API Proxy
> Documentation:
> 
>  - Latest:  
>     [`https://xkcd.now.sh/?comic=latest`](https://xkcd.now.sh/?comic=latest)
>  - Comic #303 (and so on...):  
>     [`https://xkcd.now.sh/?comic=303`](https://xkcd.now.sh/?comic=303)

However, this repo's [README](https://github.com/mrmartineau/xkcd-api/blob/91139bb33e1c8f965482cb537c0e4b93950c3403/readme.md) currently lists conflicting endpoints, which yield `404` responses:

> ## Latest comic
> ```
> https://xkcd.now.sh
> ```
> 
> ## Comic #1000
> ```
> https://xkcd.now.sh/1000
> ```

Perhaps this repo can be updated with the current project code and documentation for the live API hosted on ZEIT? The endpoints using the `comic` param seem to work just fine.

I can't see your source code for ZEIT, but I can make a PR for the README if you would like.

**Comment by mrmartineau** [comment]
Hi @jsejcksn the docs that you referenced are not actually from this project. I think someone else has created a similar project and has taken the same url as me.. I will update the docs to mention this.

**Comment by jsejcksn** [comment]
@mrmartineau Here's the relevant ZEIT documentation if you want to try to sort the domain issue:

- https://zeit.co/docs/v2/custom-domains
- https://zeit.co/docs/configuration#project/alias

I couldn't find anything in the docs that mention registration/allocation of `*.now.sh` subdomains. It seems odd to me that there could be race conditions, and, as a user of the service, I hadn't considered this event before. (I namespace my APIs according to the owner/team like `project-name.owner-or-team-name.now.sh`.)

**Comment by Smile4ever** [comment]
It seems like someone indeed made the "same" as this code. Since I'm hosting a mirror on https://xkcdapi.now.sh and the URL https://xkcdapi.now.sh/1000 works I think someone took over the xkcd.now.sh domain. 

Maybe you can take over any now.sh subdomain if you specify it as an alias?

**Comment by mrmartineau** [comment]
I had to remove a whole bunch of services after Zeit charged nearly $200 for one month last year. They changed their pricing plans after the switch from v1 to v2. The net result is that I took a bunch of stuff offline and that is when this other person must've taken the domain. I don't plan on publishing it again unless I change something, so either use @Smile4ever's mirror or host it yourself. 

I'm going to close this issue now, but let me know if you have any other questions 😄
